website/pages/docs/agent/config-entries/service-intentions.mdx

```diff
@@ -0,0 +1,310 @@
+---
+layout: docs
+page_title: 'Configuration Entry Kind: Service Intentions'
+sidebar_title: service-intentions <sup>Beta</sup>
+description: >-
+  The service-intentions config entry kind controls Connect traffic
+  authorization for both networking layer 4 (e.g. TCP) and networking layer 7
```

Comment:
Should we name the layers? Our marketing material will use phrases like "application layer" vs "transport layer", so this could be another place to improve both comprehension and SEO.

Reply:
I was saving the marketing terminology for a later pass after I get all of the main parts squared away.